rust/azure_iot_operations_services/src/schema_registry.rs

```diff
@@ -86,15 +120,16 @@ pub struct ServiceError {
     pub property_value: Option<String>,
 }
 
+// TODO: How should id be exposed, if at all?
```

Comment:
Does the GetRequest even need to be modifyable in the same way the PutRequest does? There isn't the same semantic meaning to it being something you'd save I think

Reply:
I mean, it doesn't need to be. If you'd rather I hold off on this change, I will